uelc/main/tests/test_views.py

```diff
@@ -356,3 +356,73 @@ def test_admin_edit_case_no_emptyfields(self):
     def test_admin_create_case_no_emptyfields(self):
         '''This functionality doesn't actually work...'''
         pass
+
+
+class TestAdminCohortViewContext(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.h = get_hierarchy("main", "/pages/main/")
+        self.root = self.h.get_root()
+        self.root.add_child_section_from_dict(
+            {
+                'label': 'Section 1',
+                'slug': 'section-1',
+                'pageblocks': [],
+                'children': [],
+            })
+        self.case = CaseFactory()
+        self.profile = AdminUpFactory()
+        self.gu = GroupUpFactory()
+        self.cohort = CohortFactory()
+        self.client.login(username=self.profile.user.username, password='test')
+
+    def test_uelc_admin_case_response_context(self):
+        response = self.client.get("/uelcadmin/cohort/")
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response,
+                                'pagetree/uelc_admin_cohort.html')
+        self.assertTrue(response.context['path'])
```

Comment:
If you just want to test whether a key exists in the context (or any dictionary), you could use `self.assertIn('path', response.context)`

I often forget that that one exists and just do something like `self.assertTrue('path' in request.context)`. Either way, both are a bit more precise since `self.assertTrue(response.context['path'])` would fail if the context has the `path` key, but it's set to a value that evaluates as `False`.